pkgs/development/python-modules/stable-baselines3/default.nix

```diff
@@ -0,0 +1,88 @@
+{
+  lib,
+  ale-py,
+  buildPythonPackage,
+  cloudpickle,
+  fetchFromGitHub,
+  gymnasium,
+  matplotlib,
+  numpy,
+  opencv4,
+  pandas,
+  pillow,
+  psutil,
+  pygame,
+  pytestCheckHook,
+  pythonOlder,
+  rich,
+  setuptools,
+  tensorboard,
+  torch,
+  tqdm,
+}:
+buildPythonPackage rec {
+  pname = "stable-baselines3";
+  version = "2.3.2";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "DLR-RM";
+    repo = "stable-baselines3";
+    # commit with updated dependencies since gymnasium is not compatible with the latest release:
```

Comment:
Then we should really adjust version to contain a -unstable-XXXX-XX-XX (please fill in the date of the commit)